lib/model/model.go

```diff
@@ -334,7 +334,15 @@ func (m *Model) RemoveFolder(folder string) {
 	m.pmut.Lock()
 
 	// Delete syncthing specific files
-	folderCfg := m.folderCfgs[folder]
+	folderCfg, ok := m.folderCfgs[folder]
+	if !ok {
+		// Folder might be paused
+		folderCfg, ok = m.cfg.Folder(folder)
+	}
+	if !ok {
+		// Folder does not exist... Weird...
+		return
```

Comment:
I think this should never happen and thus panic

Reply:
When/if we add `HTTP DELETE /folders/xyz` API, this would make sense.

Reply:
Shouldn't that result in a returned error? I think there are some endpoints presently that check whether a folder is present before doing anything.